ocp_resources/model_registry.py

```diff
@@ -56,10 +56,10 @@ def to_dict(self) -> None:
         super().to_dict()
 
         if not self.kind_dict and not self.yaml_file:
-            if not self.grpc:
+            if not self.grpc and self.grpc != {}:
```

Comment:
@rnetser  I think we need to handle this is the class-generator as well.

@lugi0  please do:
```
if self.grpc is None:
```

Reply:
https://github.com/RedHatQE/openshift-python-wrapper/pull/2221

Reply:
@lugi0 or better, regenerate the file after my PR is merged.

Reply:
Sounds good @myakove

Reply:
Done